helm/examples/azure/README.md

```diff
@@ -17,15 +17,15 @@ This may be okay for experimentation, where you do not care that your files are
 
 The hpcc-azurefile helm chart will provision a new Storage Class and a Persistent Volume Claim for each of the required HPCC storage types.
 Once installed the generated PVC names should be used when installing the HPCC helm chart.
-The values-retained-azurefile.yaml is an example of the HPCC storage settings that should be applied, after the "hpcc-azurefile" helm chart is installed.
+The values-retained-azurefile.yaml is an example of the HPCC storage settings that should be applied, after the "hpcc-azurefile" helm chart is installed. "azurestorage-hpcc-azurefile" name is expected when launch "helm install" to match the PVC name in the values-retained-azurefile.yaml.
 
 The storage class used in this example supports mountOptions, which are used to ensure that the mounts are owned by user and group 'hpcc' with suitable file permissions.
 
 NB: The output of installing this chart, will contain a generated example with the correct PVC names.
 
 Example use:
 
-*helm install azstorage hpcc-azurefile/*
+*helm install azstorage-hpcc-azurefile ./examples/azure/azurefile
```

Comment:
>./examples/azure/azurefile

needs to be:
./examples/azure/hpccazurefile

Reply:
still unclear why changed from "azstorage" to "azstorage-hpcc-azurefile", i.e. to include the chart name in the install name explicitly ?

The result is the same (e.g. the PVC names are the same) whether you run:
```
helm install azstorage-hpcc-azurefile ./examples/azure/azurefile
```
Or as it was (and think it still should be):
```
helm install azstorage ./examples/azure/azurefile
```

because "hpcc-azurefile.fullname" automatically the chart name.

Reply:
> change to 
```console
  helm install azstorage examples/azure/hpcc-azurefile
```
helm chart name "azstorage" with the char directory name do generate correct sc and pvc names now